src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -872,19 +907,17 @@ private static bool TryGetEasyRequestFromContext(IntPtr context, out EasyRequest
                 return false;
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null)
+            private void EventSourceTrace<TArg0>(string formatMessage, TArg0 arg0, EasyRequest easy = null, [CallerMemberName] string memberName = null)
```

Comment:
It's subtle, but these overloads fill in the agent automatically (see the overload they call below).  That requires they be instance methods.  Alternatively, I could remove all of these and at all of the call sites explicitly pass in ```this```.  I could do that if you'd prefer...?